src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/InterchangeHeaderTest.java

```diff
@@ -80,6 +80,7 @@ void testFromStringWithValidEdifactStringReturnsInterchangeHeader() {
         assertEquals("SNDR", interchangeHeader.getSender());
         assertEquals("RECP", interchangeHeader.getRecipient());
         assertEquals(1L, interchangeHeader.getSequenceNumber());
+        assert (!interchangeHeader.isNhsAckRequested());
```

Comment:
Let's stick to framework assertions
```suggestion
        assertFalse(interchangeHeader.isNhsAckRequested());
```
Here and other tests